main/helpers.py

```diff
@@ -0,0 +1,38 @@
+from .models import Player, Field, Match
+
+
+def get_invalid_matches():
+    matches = Match.objects.all()
+    invalid_match_ids = [match.id for match in matches if is_match_invalid(match)]
+    return invalid_match_ids
+
+
+def is_match_invalid(match):
+    score_list = [match.player2_score, match.player1_score]
+    if score_list.count(11) == 1:
+        if 0 <= match.player1_score <= 11 and 0 <= match.player2_score <= 11:
+            return False
+        else:
+            return True
+    else:
+        return True
+
+
+def remove_invalid_matches():
+    invalid_match_ids = get_invalid_matches()
+    for match_id in invalid_match_ids:
+        Match.objects.get(pk=match_id).delete()
```

Comment:
Minor: I'd try-catch here, due to concurrency reasons (e.g. two requests being processed at the same time) there's no guarantee that match.objects.get will return anything / not raise an exception. You currently call delete() on a dubious object

Reply:
I'd say this is important for @thomasbean-prezi to understand what's the caveat here, but not super important to guard against this here.

Reply:
I thought putting try catch blocks around the points when I am getting objects from the models, and I do see the need for it. I ultimately did not because I thought this would only be running locally and I would be the only person modifying the data.
Thank you for pointing it out @ochronus , if i end up deploying this to heroku or something like that I will implement try catch blocks where necessary